run.go

```diff
@@ -190,6 +194,16 @@ func runff(ff *funcfile, stdin io.Reader, stdout, stderr io.Writer, method strin
 		}
 	}
 
+	if !rwRoot {
+		sh = append(sh, "--read-only")
+	}
+
+	if ff.TmpFsSize != 0 {
```

Comment:
Let's always try to add a rw tmpfs here. This is also the default behavior in fn agent now.